js/src/features/project/components/Team/TeamChangeStatus.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Button } from "antd";
+
+import "./TeamChangeStatus.scss";
+
+type Props = {
+  onChange: () => void;
```

Comment:
And again, maybe better `onButtonClick`?

Reply:
fix